packages/vector_graphics/lib/vector_graphics.dart

```diff
@@ -140,40 +146,43 @@ class _VectorGraphicsWidgetState extends State<VectorGraphic> {
   @override
   Widget build(BuildContext context) {
     final PictureInfo? pictureInfo = _pictureInfo;
-    if (pictureInfo == null) {
-      return SizedBox(width: widget.width, height: widget.height);
-    }
-
-    // If the caller did not specify a width or height, fall back to the
-    // size of the graphic.
-    // If the caller did specify a width or height, preserve the aspect ratio
-    // of the graphic and center it within that width and height.
-    double? width = widget.width;
-    double? height = widget.height;
-
-    if (width == null && height == null) {
-      width = pictureInfo.size.width;
-      height = pictureInfo.size.height;
-    } else if (height != null && !pictureInfo.size.isEmpty) {
-      width = height / pictureInfo.size.height * pictureInfo.size.width;
-    } else if (width != null && !pictureInfo.size.isEmpty) {
-      height = width / pictureInfo.size.width * pictureInfo.size.height;
-    }
 
-    Widget child = SizedBox(
-      width: width,
-      height: height,
-      child: FittedBox(
-        fit: widget.fit,
-        alignment: widget.alignment,
-        child: SizedBox.fromSize(
-          size: pictureInfo.size,
-          child: _RawVectorGraphicsWidget(
-            pictureInfo: pictureInfo,
+    Widget child;
+    if (pictureInfo != null) {
+      // If the caller did not specify a width or height, fall back to the
+      // size of the graphic.
+      // If the caller did specify a width or height, preserve the aspect ratio
+      // of the graphic and center it within that width and height.
+      double? width = widget.width;
+      double? height = widget.height;
+
+      if (width == null && height == null) {
+        width = pictureInfo.size.width;
+        height = pictureInfo.size.height;
+      } else if (height != null && !pictureInfo.size.isEmpty) {
+        width = height / pictureInfo.size.height * pictureInfo.size.width;
+      } else if (width != null && !pictureInfo.size.isEmpty) {
+        height = width / pictureInfo.size.width * pictureInfo.size.height;
+      }
+
+      child = SizedBox(
+        width: width,
+        height: height,
+        child: FittedBox(
+          fit: widget.fit,
+          alignment: widget.alignment,
+          child: SizedBox.fromSize(
+            size: pictureInfo.size,
+            child: _RawVectorGraphicsWidget(
+              pictureInfo: pictureInfo,
+            ),
           ),
         ),
-      ),
-    );
+      );
+    } else {
+      child = widget.placeholderBuilder(context);
```

Comment:
in flutter_svg this returns a SizedBox with the width/height if it's not null.

I don't think the limited box part is important here - we could just unconditionally return the sized box. I just wanted it to be something "as cheap as possible" in the case where no width/height had been specified.

Reply:
IOW I think something like this would work
```suggestion
      child = widget.placeholderBuilder?.call(context) ?? SizedBox(width: widget.width, height: widget.height);
```

Reply:
Ahh, I was actually looking the avd example. Will update the builder. Trying to keep signature identical

Reply:
Done